checker/check-api-tag-removed.go

```diff
@@ -0,0 +1,43 @@
+package checker
+
+import (
+	"fmt"
+
+	"github.com/tufin/oasdiff/diff"
+)
+
+const (
+	apiTagRemovedCheckId = "api-tag-removed"
+)
+
+func APITagRemovedCheck(diffReport *diff.Diff, operationsSources *diff.OperationsSourcesMap, config BackwardCompatibilityCheckConfig) []BackwardCompatibilityError {
+	result := make([]BackwardCompatibilityError, 0)
+	if diffReport.PathsDiff == nil {
+		return result
+	}
+
+	for path, pathItem := range diffReport.PathsDiff.Modified {
+		if pathItem.OperationsDiff == nil {
+			continue
+		}
+
+		for operation, operationItem := range pathItem.OperationsDiff.Modified {
+			op := pathItem.Base.Operations()[operation]
+			source := (*operationsSources)[op]
+
+			if operationItem.TagsDiff == nil || len(operationItem.TagsDiff.Deleted) == 0 {
+				continue
+			}
+
+			result = append(result, BackwardCompatibilityError{
+				Id:        apiTagRemovedCheckId,
+				Level:     ERR,
+				Text:      fmt.Sprintf(config.i18n(apiTagRemovedCheckId), ColorizedValue(operationItem.TagsDiff.Deleted)),
```

Comment:
The error message tries to print a string, but operationItem.TagsDiff.Deleted is a StringList:
error	[api-tag-removed] at data/simple1.yaml
	in API GET /api/test
		api tag '[a b]' removed

Perhaps it would be better to print an error per deleted tag, or to change the error to read:
error	[api-tag-removed] at data/simple1.yaml
	in API GET /api/test
		api tags 'a', 'b' removed

Reply:
Great point! Going w/ option 1: error per deleted tag.